src/Kernel-BytecodeEncoders/EncoderForSistaV1.class.st

```diff
@@ -795,6 +795,24 @@ EncoderForSistaV1 class >> pushNilByte [
 	^ 79
 ]
 
+{ #category : #'compiled method support' }
+EncoderForSistaV1 class >> readsSelfFor: compiledMethod [
```

Comment:
There is #in: end in similar methods. And it sounds better: #readsSelfIn: and readsThisContextIn:

Reply:
I made it follow the existing isSuperSendFor: method...

Reply:
The best here would be to have just the block on the encoder and do the scanning on the method in CompiedCode. Most other scanning methods do that. I have added that to the backlog